docs/tutorials/open-source-frameworks/apache-flink/0_setup.qmd

```diff
@@ -0,0 +1,53 @@
+# Getting started
+
+In this tutorial, you will learn how to set up and use Flink with Ibis. If you
+want to see Ibis in action with a real-life example, check out [A real-life
+use case: fraud detection](1_single_feature.qmd).
```

Comment:
I would slightly rephase the second sentence to something like "after setup in this part, we'll see a real-life example..." or something just so the user doesn't jump ahead before getting everything installed and whatnot